test/image/mocks/funnel_axis.json

```diff
@@ -0,0 +1,143 @@
+                2,
+                1
+            ],
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x",
+            "yaxis": "y"
+        },
+        {
+            "type": "funnel",
+            "orientation": "v",
+            "x": [
+                "A",
+                "B",
+                "C"
+            ],
+            "y": [
+                300,
+                200,
+                100
+            ],
+            "textposition": "inside",
+            "insidetextanchor": "start",
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x2",
+            "yaxis": "y2"
+        },
+        {
+            "type": "funnel",
+            "orientation": "h",
+            "x": [
+                30000,
+                20000,
+                10000
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "textposition": "inside",
+            "insidetextanchor": "end",
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x3",
+            "yaxis": "y3"
+        },
+        {
+            "type": "funnel",
+            "orientation": "h",
+            "x": [
+                3000000,
+                2000000,
+                1000000
+            ],
+            "y": [
+                "A",
+                "B",
+                "C"
+            ],
+            "textposition": "outside",
+            "cliponaxis": false,
+            "textinfo": "value+percent initial+percent previous+percent total",
+            "xaxis": "x4",
+            "yaxis": "y4"
+        }
+    ],
+    "layout": {
+        "width": 800,
+        "height": 800,
+        "dragmode": "pan",
+        "xaxis": {
+            "type": "category",
+            "domain": [
+                0,
+                0.48
+            ],
+            "autorange": true
+        },
+        "xaxis2": {
+            "type": "category",
+            "domain": [
+                0.52,
+                1
+            ],
+            "autorange": "reversed"
+        },
+        "xaxis3": {
+            "domain": [
+                0,
+                0.48
+            ],
+            "autorange": true
+        },
+        "xaxis4": {
+            "domain": [
+                0.52,
+                1
+            ],
+            "autorange": "reversed"
+        },
+        "yaxis": {
+            "domain": [
+                0,
+                0.48
+            ],
+            "autorange": "reversed"
+        },
+        "yaxis2": {
+            "domain": [
+                0.52,
+                1
+            ],
+            "autorange": true
+        },
+        "yaxis3": {
+            "type": "category",
+            "domain": [
+                0.52,
+                1
+            ],
+            "autorange": "reversed"
+        },
+        "yaxis4": {
```

Comment:
Oh, when dragging this subplot

![Peek 2019-04-29 17-41](https://user-images.githubusercontent.com/6675409/56928869-191da780-6aa6-11e9-8f04-d91a59e99053.gif)

I get:

![image](https://user-images.githubusercontent.com/6675409/56928840-099e5e80-6aa6-11e9-8888-3ee9f0af72c9.png)

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/3817/commits/04049f2ca3c440d4e4da429b1481f58cdd660daa .

Reply:
Thanks!